### ISSUE TITLE
Request logging CLI option reported as unknown

### ISSUE THREAD

**Issue by ssams** [vercel/serve#744]
### Description

the CLI utility lists a "--no-request-logging" option (shorthand "-L") in its "--help" output, which does not work. The CLI fails to start and reports the parameter as unknown if it's present on the command line:

`npx serve@14.1.1 --no-request-logging dist/`

error:
```
/root/.npm/_npx/abcb09c5f4b4e0d1/node_modules/arg/index.js:132
                                                throw new ArgError(
                                                      ^

ArgError: unknown or unexpected option: --no-request-logging
    at arg (/root/.npm/_npx/abcb09c5f4b4e0d1/node_modules/arg/index.js:132:13)
    at parseArguments (file:///root/.npm/_npx/abcb09c5f4b4e0d1/node_modules/serve/build/main.js:390:28)
    at file:///root/.npm/_npx/abcb09c5f4b4e0d1/node_modules/serve/build/main.js:467:40
    at ModuleJob.run (node:internal/modules/esm/module_job:193:25)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:526:24)
    at async loadESM (node:internal/process/esm_loader:91:5)
    at async handleMainPromise (node:internal/modules/run_main:65:12) {
  code: 'ARG_UNKNOWN_OPTION'
}
```

looks like the option is only present in the static help text in https://github.com/vercel/serve/blob/aa4a82761eda14630868609ca93ccb035421c3e9/source/utilities/cli.ts#L47, but missing in the actual parsed options in https://github.com/vercel/serve/blob/aa4a82761eda14630868609ca93ccb035421c3e9/source/utilities/cli.ts#L146?

### Library version

14.1.1

### Node version

v16.18.1

**Comment by gamemaker1** [comment]
Oops yes it is missing, let me fix that :)

**Comment by BasitAliAhmad** [comment]
When this will be fixed? @gamemaker1

**Comment by livinglogic-nl** [comment]
Can somebody please merge #742 ?

We use serve in our pipelines and our build logs are now flooded with log messages.
#742 semi-fixes this by at least allowing us to turn off the verbose logging.

It seems weird that people are apparently forking this project to fix the issue when this PR could just be approved...

**Comment by AndyBitz** [comment]
Thank you for opening the issue, the PR with the fix https://github.com/vercel/serve/pull/742 has been merged and is part of the latest release 14.1.2
